packages/manager/core/generator/app/conditional-templates/onboarding/onboarding.test.ts.hbs

```diff
@@ -1,6 +1,6 @@
 import { expect, test } from '@playwright/test';
-import '../../../../../playwright/login';
-import * as config from '../../../../../playwright/config.json';
+import '@/../../playwright/login';
+import * as config from '@/../../playwright/config.json';
```

Comment:
Maybe like sugested here : https://github.com/ovh/manager/pull/9447/files#r1238703007
An alias for `@playwright-helpers/config.json`